--- a/vendor.qti.adsprpc-service.rc
+++ b/vendor.qti.adsprpc-service.rc
@@ -35,6 +35,7 @@ service vendor.adsprpcd /vendor/bin/adsprpcd
    class main
    user system
    group media
+   setenv ADSP_LIBRARY_PATH "/vendor/lib/rfsa/adsp;/odm/lib/rfsa/adsp;/system/vendor/lib/rfsa/adsp;/vendor/dsp"
 
 service vendor.adsprpcd_audiopd /vendor/bin/adsprpcd audiopd
    class main
@@ -42,5 +43,5 @@ service vendor.adsprpcd_audiopd /vendor/bin/adsprpcd audiopd
    group media
    #ifdef OPLUS_FEATURE_KTV
    #Juqiang.Geng@MULTIMEDIA.AUDIODRIVER.ADSP,2020/09/22, Add ADSP_LIBRARY_PATH new path
-   setenv ADSP_LIBRARY_PATH ";/system/lib/rfsa/adsp;/system/vendor/lib/rfsa/adsp;/dsp;/vendor/dsp;/odm/lib/rfsa/adsp;/vendor/lib/rfsa/adsp"
-   #endif /* OPLUS_FEATURE_KTV */
\ No newline at end of file
+   setenv ADSP_LIBRARY_PATH "/vendor/lib/rfsa/adsp;/odm/lib/rfsa/adsp;/system/vendor/lib/rfsa/adsp;/vendor/dsp"
+   #endif /* OPLUS_FEATURE_KTV */
